PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -522,6 +522,9 @@
                 Plot1D('mvaLowPt', 'mvaLowPt', 20, -1, 1, 'Low pt muon ID score'),
                 Plot1D('mvaLowPtId', 'mvaLowPtId', 3, -0.5, 2.5, 'Low Pt Mva ID from miniAOD selector (1=LowPtMvaLoose, 2=LowPtMvaMedium)'),
                 Plot1D('mvaTTH', 'mvaTTH', 20, -1, 1, 'TTH MVA lepton ID score'),
+                Plot1D('mvaIDMuon', 'mvaIDMuon', 20, -1, 1, 'Score of MVA-based muon ID'),
+                Plot1D('mvaIDMuon_WP', 'mvaIDMuon_WP',  3, -0.5, 2.5, 'MVA-based ID selector WPs (1=MVAIDwpMedium,2=MVAIDwpTight)'),
+                Plot1D('mvaIDMuon_nano', 'mvaIDMuon_nano', 20, -1, 1, 'MVA-based ID score (from nanoAOD)'),
```

Comment:
- The `mvaIDMuon_nano` is not present anymore since it's now `mvaIDMuon` whether or not it's taken from mini or nano
- If the `mvaIDMuon_WP` variable is only present in some eras and not others, it should be removed using modifiers in https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/nanoDQM_cff.py